frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -434,22 +433,29 @@ class BitBox02 extends Component<Props, State> {
                     )
                 }
                 <div className="container">
-                    <Header title={<h2>{t('welcome.title')}</h2>}>
-                        <LanguageSwitch />
-                    </Header>
+                    <Status hidden={attestationResult !== false}>
+                        {t('bitbox02Wizard.attestationFailed')}
+                    </Status>
```

Comment:
Moved the attestation warning to the top, but on the unlock view it will be covered by the Fullscreen component, that is why added it there as well.